ParkingZoneApp/Areas/Admin/Controllers/ParkingSlotController.cs

```diff
@@ -61,4 +62,52 @@ public IActionResult Create(CreateViewModel createModel)
         return View(createModel);
     }
     #endregion
+
+    #region Edit
+    // GET: Admin/ParkingSlot/Edit/5
+    public IActionResult Edit(int? id)
+    {
+        var slot = _slotService.GetById(id);
+        if (slot == null)
+        {
+            return NotFound();
+        }
+        var editModel = new EditViewModel(slot);
+        return View(editModel);
+
+    }
+
+    // POST: Admin/ParkingSlot/Edit/5
+    [HttpPost]
+    [ValidateAntiForgeryToken]
+    public IActionResult Edit(int id, EditViewModel editModel)
```

Comment:
In this action you are not checking if the submitted ParkingSlot.Number is the same. 
Aytmoqchimanki agar Editda ParkingSlot Numberni uzgartirmasdan Submit qsez Exist db cb qolad uzinikidaqa qomed